thorlcr/thorutil/thormisc.cpp

```diff
@@ -76,7 +76,7 @@ const StatisticsMapping basicActivityStatistics({StTimeLocalExecute, StTimeBlock
 const StatisticsMapping groupActivityStatistics({StNumGroups, StNumGroupMax}, basicActivityStatistics);
 const StatisticsMapping hashJoinActivityStatistics({StNumLeftRows, StNumRightRows}, basicActivityStatistics);
 const StatisticsMapping indexReadActivityStatistics({StNumRowsProcessed, StNumIndexSeeks, StNumIndexScans, StNumPostFiltered, StNumIndexWildSeeks}, basicActivityStatistics);
-const StatisticsMapping indexWriteActivityStatistics({StPerReplicated}, basicActivityStatistics);
+const StatisticsMapping indexWriteActivityStatistics({StPerReplicated, StNumDiskWrites}, basicActivityStatistics);
```

Comment:
change to

    const StatisticsMapping indexWriteActivityStatistics({StPerReplicated}, basicActivityStatistics, diskWriteRemoteStatistics)

similar to the indexWriteActivityStats.